modules/api/static/schema.yml

```diff
@@ -0,0 +1,1916 @@
+      responses:
+        '501':
+          description: This instrument flags values
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+          
+    put:
+      tags:
+        - Instruments
+      summary: Set an instrument flags values
+      parameters:
+        - name: candid
+          in: path
+          description: The candidate identifier
+          required: true
+          schema:
+            type: string
+        - name: visit
+          in: path
+          description: The visit label
+          required: true
+          schema:
+            type: string
+        - name: instrument
+          in: path
+          description: The instrument short name
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InstrumentFlags'
+      responses:
+        '501':
+          description: This instrument flags values
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+          
+  '/candidates/{candid}/{visit}/instruments/{instrument}/dde':
+    get:
+      tags:
+        - Instruments
+      summary: Get an instrument data values
+      parameters:
+        - name: candid
+          in: path
+          description: The candidate identifier
+          required: true
+          schema:
+            type: string
+        - name: visit
+          in: path
+          description: The visit label
+          required: true
+          schema:
+            type: string
+        - name: instrument
+          in: path
+          description: The instrument short name
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: This instrument data values
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Instrument'
+                    
+    patch:
+      tags:
+        - Instruments
+      summary: Update some data values of an instrument
+      parameters:
+        - name: candid
+          in: path
+          description: The candidate identifier
+          required: true
+          schema:
+            type: string
+        - name: visit
+          in: path
+          description: The visit label
+          required: true
+          schema:
+            type: string
+        - name: instrument
+          in: path
+          description: The instrument short name
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Instrument'
+      responses:
+        '204':
+          description: This instrument data values have been saved
+          
+    put:
+      tags:
```

Comment:
input boxes and request. Maybe just pick the request which is more complete?

Also, does not execute for some reason

Reply:
working now.